src/main/webapp/index.html

```diff
@@ -33,7 +33,13 @@ <h1>PIECED</h1>
             With Pieced, we aimed to bring the concept of visualizing animal species through their population to a larger scale, providing additional information about the species along the way.
             <br>
             <br>
-            <small class="text-muted">Created by Fiza Goyal, Ariana Zhu, and Maxwell Chen for Google's 2020 STEP internship program.</small>
+            <small class="text-muted">Created by Fiza Goyal, Ariana Zhu, and Maxwell Chen for the Google STEP internship program in Summer 2020. They were hosted by Tom Russ and Dave Newman.</small>
+            <br>
+            <small class="text-muted">
+                <a href="https://github.com/fgoyal"> Fiza</a> is a rising junior ('22) at University of Illinois at Urbana-Champaign.
+                <a href="https://github.com/arianazhu"> Ariana<a> is a rising junior ('22) at Purdue University.
+                <a href="https://github.com/maxhchen"> Maxwell</a> is a rising junior ('22) at UC Berkeley.
```

Comment:
Super minor but do you think it would flow better if University of California, Berkeley was written out? I'm ok with it either way though!

Reply:
Sure, will update

Reply:
Hmm actually, changing that pushes the text onto two lines, so I think I'll leave it at UC Berkeley

Reply:
Nevermind, found a workaround